ansible/playbook.yml

```diff
@@ -179,10 +179,10 @@
   - role: gcsfuse
     when:
     - install_gcsfuse
-    - ansible_architecture == "x86_64"
+    - ansible_architecture == "x86_64" or (ansible_distribution == "Ubuntu" and ansible_distribution_version is version('24.04', '=='))
   - role: cloudagents
     when:
-    - ansible_architecture == "x86_64"
+    - ansible_architecture == "x86_64" or (ansible_distribution == "Ubuntu" and ansible_distribution_version is version('24.04', '=='))
```

Comment:
A similar statement holds here:

https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent#oagent-arm-support

I would review this documentation with the goal of eliminating the `when` entirely. If it breaks on something EOL (e.g. CentOS 7) that's fine.